packages/core/forms/src/forms/RLAForm.vue

```diff
@@ -0,0 +1,428 @@
+<template>
+  <div class="rla-form-basic-fields">
+    <VueFormGenerator
+      :model="formModel"
+      :options="formOptions"
+      :schema="scopingSchema"
+      @model-updated="(value: any, model: string) => onModelUpdated(value, model)"
+    />
+
+    <KCard
+      class="rla-form-request-limits"
+    >
+      <template #title>
+        <KLabel
+          class="rla-form-request-limits-title"
+          for="rla-form-request-limits-legend"
+          :info="t('rla.request_limits.help')"
+          required
+          :tooltip-attributes="{
+            maxWidth: '300',
+            placement: 'top',
+            tooltipId: 'rla-form-request-limits-tooltip'
```

Comment:
I find assigning `tooltipId` here has no effect because it'll be overwritten by https://github.com/Kong/kongponents/blob/76b6d34c7b01323e37a437e1b4378d9f40e0f5ae/src/components/KLabel/KLabel.vue#L14

Reply:
@Leopoldthecoder Thanks. I just removed that property.